tools/libcloud/config.py

```diff
@@ -1,101 +1,117 @@
 import os
 import logging
 import argparse
+import json
 
-# cloudinit
-BASE_CLOUD_INIT_PATH = os.path.join(
-    os.path.dirname(os.path.realpath(__file__)),
-    "..",
-    "cloudinit")
-
-# default credentials
-DEFAULT_EMAIL = 'libcloud@fast-gateway-233909.iam.gserviceaccount.com'
-DEFAULT_CREDS = '~/.gcloud.json'
-PROJECT = os.getenv('RADIX_MTPS_CLOUD_PROJECT', 'fast-gateway-233909')
-
-# network config
-DEFAULT_NETWORK_UNIVERSE = ""
-DEFAULT_NETWORK_PASSWORD = "DEFAULT_PASSWORD"
-DEFAULT_NETWORK_ATOMS_FILE = "/radix/atoms_v4_full.zst"
-DEFAULT_NETWORK_START_PUMPING = 40  # X minutes
-DEFAULT_NETWORK_SHARD_COUNT = "1181"
-DEFAULT_NETWORK_SHARD_OVERLAP = "0.1"
+STORAGE = {
```

Comment:
Could this be called `CONFIG`?

Reply:
if called CONFIG then reference to that will be like config.CONFIG['VALUENAME']